File: AppTour.Android/src/com/apptour/framework/UserModel.java
package com.apptour.framework;

import java.util.Date;

public class UserModel {

	private static UserModel instance;
	
	public String Id;
    public boolean IsActive;
    public String UserName;
    public String Password;
    public String NewPassword;
    public String Email;
    public String RealName;
    public String Role;
    public Date CreationDate;    
    public Date UpdateDate;

    // Singleton Pattern for User 
    public static UserModel getInstance(){
    	if (instance==null){
    		instance = new UserModel();
    	}
    	return instance;
    }
    
}

File: AppTour.System/AppTour.Model.Models/User/UserModel.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;
using AppTour.Model.Models.Helpers;

namespace AppTour.Model.Models.User
{
    [DataContract]
    public class UserModel : BaseModel
    {
        public UserModel() { }

        [DataMember]
        public Guid Id { get; set; }

        [DataMember]
        public string UserName { get; set; }

        [DataMember]
        public string Password { get; set; }

        [DataMember]
        public string NewPassword { get; set; }

        [DataMember]
        public string Email { get; set; }

        [DataMember]
        public string RealName { get; set; }

        [DataMember]
        public string Role { get; set; }

        [DataMember]
        [DataType(DataType.Date)]
        public DateTime CreationDate { get; set; }

        [DataMember]
        public DateTime UpdateDate { get; set; }

        [DataMember]
        public bool IsActive { get; set; }

        public override FluentValidation.Results.ValidationResult SelfValidate()
        {
            return ValidationHelper.Validate<UserValidator, UserModel>(this);
        }
    }
}
